### PR TITLE
Ki/APPEALS-17507 & 17520 - Edit Modal Created, Connect Dropdown and Modal

### PR DESCRIPTION
**Pull request by KiMauVA**
Resolves #{17507 & 17520}
[APPEALS-17507](https://vajira.max.gov/browse/APPEALS-17507)
[APPEALS-17520](https://vajira.max.gov/browse/APPEALS-17520)

### Description
This includes both the Edit Modal work, as well as the functionality to connect it to the Edit Issue dropdown.

### Acceptance Criteria
- [X] Code compiles correctly

### Testing Plan
[APPEALS-21345](https://vajira.max.gov/browse/APPEALS-21345)

### User Facing Changes
 - [X] Screenshots of UI changes added to Original Issue JIRA Ticket [APPEALS-17507](https://vajira.max.gov/browse/APPEALS-17507)

### CODE REVIEW

**Comment by KiMauVA** [comment]
Screen Shot 2023-04-21 at 2 43 04 PM
I did do a merge for the feature, but for some reason additional files are showing up that already exist on the feature branch, like the Badges and other work.

File: client/COPY.json [hunk 1]
```diff
@@ -1042,7 +1042,15 @@
...
-  "INTAKE_VHA_CLAIM_REVIEW_REQUIREMENT": "Only VHA team members can establish Higher Level Reviews and Supplemental Claims with VHA issues. If you have a VHA claim, please return the packet to the main “VHA” queue in the Centralized Mail Portal or send downloaded documents to %s.",
+  "INTAKE_EDIT_ISSUE_TITLE": "Edit issue",
+  "INTAKE_EDIT_ISSUE_SELECT_SPECIAL_ISSUES": "Select any special issues that apply",
+  "INTAKE_EDIT_ISSUE_CHANGE_MESSAGE": "Why was this change made?",
+  "INTAKE_EDIT_ISSUE_LABEL": "Issue: ",
+  "INTAKE_EDIT_ISSUE_BENEFIT_TYPE": "Benefit type: ",
+  "INTAKE_EDIT_ISSUE_DECISION_DATE": "Decision date: ",
+  "INTAKE_EDIT_ISSUE_MST_LABEL": "Military Sexual Trauma (MST)",
+  "INTAKE_EDIT_ISSUE_PACT_LABEL": "PACT Act",
+  "INTAKE_VHA_CLAIM_REVIEW_REQUIREMENT": "Only VHA team members can establish High Level Reviews and Supplemental Claims with VHA issues. If you have a VHA claim, please return the packet to the main “VHA” queue in the Centralized Mail Portal or send downloaded documents to %s.",
```
Comment (HunJerBAH) [inline client/COPY.json:1053]: Reverting this line back to original string value listed on line 1045.

File: client/app/intake/components/RemoveIssueModal.jsx [hunk 2]
```diff
@@ -34,6 +34,8 @@ class RemoveIssueModal extends React.PureComponent {
     } = this.props;
 
     return <div className="intake-remove-issue">
+      {console.log(`wut is this: ${JSON.stringify(removeIndex)}`)}
```
Comment (HunJerBAH) [inline client/app/intake/components/RemoveIssueModal.jsx:37]: removing this console log

**Review by HunJerBAH — APPROVED** [review]
Great work!

**Review by HunJerBAH — APPROVED** [review]

**Pull request merged**